### ISSUE TITLE
Order of breakpoints

### ISSUE THREAD

**Issue by Joobs** [maizzle/framework#36]
- Maizzle Version: 0.1.2
- Node.js Version: 10.16.0

### Description:
By default breakpoints use max-width setting (desktop first) methodology, however the CSS order is set to min-width order/methodology. When using several breakpoints on an element, the CSS order means only the largest breakpoint is used (excluding full).

### Steps To Reproduce:
```
<div class="xs-w-128 sm-w-256 w-full bg-black text-white">
text
</div>
```
In the above example the DIV will never be 128, as the 256 rule will always override it.

### Possible solution:
Changing the breakpoint order fixed the issue for me:
BEFORE
```
screens: {
      all: {'raw': 'screen'},
      xs: {'max': '480px'},
      sm: {'max': '600px'}
 },
```
AFTER
```
screens: {
      all: {'raw': 'screen'},
      sm: {'max': '600px'},
      xs: {'max': '480px'}
 },
```

**Comment by cossssmin** [comment]
Hi, can you please run `npm list @maizzle/framework` in your project, to check the framework version? (`maizzle -v` actually reports the CLI version).

I've tested this by spinning up a fresh Maizzle project (`maizzle new myproject`) and creating a template with that code:

```html
{% block template %}

<div class="xs-w-128 sm-w-256 w-full bg-black text-white">Test</div>

{% endblock %}
```

That outputs:

```html
<!DOCTYPE html>
<html lang="en" xmlns:v="urn:schemas-microsoft-com:vml" xmlns:o="urn:schemas-microsoft-com:office:office">
<head>
  <meta charset="utf8">
  <meta http-equiv="x-ua-compatible" content="ie=edge">
  <meta name="viewport" content="width=device-width,initial-scale=1">
  <meta name="x-apple-disable-message-reformatting">

  <!--[if mso]>
  <xml>
    <o:OfficeDocumentSettings>
      <o:PixelsPerInch>96</o:PixelsPerInch>
    </o:OfficeDocumentSettings>
  </xml>
  <style>
    table {border-collapse: collapse;}
    td,th,div,p,a,h1,h2,h3,h4,h5,h6 {font-family: "Segoe UI", sans-serif; mso-line-height-rule: exactly;}
  </style>
  <![endif]-->
  <style>
    @media screen {
      img {
        max-width: 100%;
      }
    }
    @media (max-width: 600px) {
      .sm-w-256 {
        width: 256px !important;
      }
    }
    @media (max-width: 480px) {
      .xs-w-128 {
        width: 128px !important;
      }
    }
  </style>
</head>
<body style="margin: 0; padding: 0; width: 100%; word-break: break-word; -webkit-font-smoothing: antialiased;">
  <div class="xs-w-128 sm-w-256" style="background-color: #000000; color: #ffffff; width: 100%;">asd</div>
</body>
</html>
```

... which if you open in a browser, you'll see it works as intended: on the `480px` breakpoint, the `128px` width is used, then for the  `600px` breakpoint `256px`, and by default it's full width.

Not sure why changing the order would fix that, maybe it's a (n old) Tailwind issue, actually? What Tailwind version are you running? (run `npm list tailwindcss`)

**Comment by Joobs** [comment]
@maizzle/framework@0.3.1
tailwindcss@1.1.2

I've created a new project, after experimenting the issue I was having was based in the local build.

`maizzle build local` - the `600px` breakpoints are build after the `480px` breakpoints.
`maizzle build production` - works as expected, `600px` breakpoints are before the `480px` breakpoints.

**Comment by cossssmin** [comment]
I see it now, you're right, it's only affecting the local builds because for _other_ build enviroments `css-mqpacker` is used and it re-orders the media queries top down from largest breakpoint.

I think I'm going to enable the `css-mqpacker` for local builds as well, so that you don't have to worry about the order of your breakpoints in your Tailwind config.

Thanks for spotting this! 👍

**Comment by cossssmin** [comment]
Just released this in [`v0.3.2` ](https://github.com/maizzle/framework/releases/tag/v0.3.2)
